src/snorkel_extractor_memex/dataset_utils.py

```diff
@@ -169,7 +193,7 @@ def clean_extracted_text(text):
     text = re.sub(r'\\x[a-zA-Z0-9][a-zA-Z0-9]', '',text)
     # Removing internal double quotes
     text = replace_middle_double_quotes(text)
-    
+    text = replace_middle_double_quotes(text)
```

Comment:
Yeah, because that regex grabs up to 2 characters ahead of the double quote, it's possible that it could miss a double quote the first time through and cause a malformatted json error. If there's a better way to do that, I'm all ears.